frontend/src/components/common/DbLanguageChip.tsx

```diff
@@ -1,20 +1,20 @@
 import React, { useCallback } from "react";
 import type { Theme } from "@mui/material";
 import { Chip, useTheme } from "@mui/material";
-import { SourceLanguage } from "@utils/constants";
+import { DbLanguage } from "@utils/api/types";
 
-const getLanguageColor = (language: SourceLanguage, theme: Theme): string => {
+const getLanguageColor = (language: DbLanguage, theme: Theme): string => {
   switch (language) {
-    case SourceLanguage.PALI: {
+    case "pa": {
```

Comment:
don't we want to use these enums anymore?

Reply:
Please see PR description.

Reply:
@aminahbl I see that, but can't we use the new `DbLanguage` object here then?

Reply:
From my point of view, there's no need and it's preferable to be trim. But my working assumption, is that all IDEs give autocomplete and/or type support so that it's extremely clear what is available and impossible to enter an invalid value, even without running tsc.

![image](https://github.com/user-attachments/assets/10641e01-e215-47c3-8f13-7ed9d635ffd9)

![image](https://github.com/user-attachments/assets/d939bdbd-1c86-4c90-bf7c-a6ca046f871d)

Glad to hear the counter view.

Reply:
this is perfect, let's leave it like this! Great that it's strongly typed